plugins/caches.js

```diff
@@ -91,7 +91,9 @@ exports.plugin = {
                 // for old json files, the value is hidden in an object, we cannot stream it directly
                 if (usingS3 && cacheName.endsWith('.zip')) {
                     try {
-                        value = await awsClient.getDownloadStream({ cacheKey });
+                        const { s3Stream } = await awsClient.getDownloadStream({ cacheKey });
+
+                        value = s3Stream;
```

Comment:
do you need to add content length here?

Reply:
eventually yes, when we remove the `zip` check in line 92 above, and test the flow for caches. For now not changing a path which is out of scope for this change.